junit-runtime/src/main/scala/scala/scalanative/junit/Reporter.scala

```diff
@@ -102,9 +102,10 @@ private[junit] final class Reporter(eventHandler: EventHandler,
       ""
     }
 
-    val m = formatTest(method, Ansi.RED)
+    val m             = formatTest(method, Ansi.RED)
+    val timeFormatted = f"$timeInSeconds%.3f"
     val msg =
-      s"$prefix$m failed: $fmtName${ex.getMessage}, took $timeInSeconds sec"
+      s"$prefix$m failed: $fmtName${ex.getMessage}, took $timeFormatted sec"
```

Comment:
Why not
```suggestion
      f"$prefix$m failed: $fmtName${ex.getMessage}, took $timeInSeconds%.3f sec"
```
?

Reply:
I'm not too familiar with `f' Interpolator and thought I'd need to add `%s` for each other usage. Great to know it can be used that way. Thank you for showing me that.